core/server/src/main/java/alluxio/worker/block/BlockWorkerClientRestServiceHandler.java

```diff
@@ -289,16 +288,15 @@ public Response readBlock(@QueryParam("sessionId") Long sessionId,
   @POST
   @Path(REQUEST_BLOCK_LOCATION)
   @Produces(MediaType.APPLICATION_JSON)
-  @ReturnType("java.lang.Long")
```

Comment:
this is an annotation for the REST API documentation ... so it was just incorrectly annotated